javascript/ql/src/Security/CWE-020/IncompleteUrlSubstringSanitization.ql

```diff
@@ -15,7 +15,7 @@ private import semmle.javascript.dataflow.InferredTypes
 
 from DataFlow::MethodCallNode call, string name, DataFlow::Node substring, string target
 where
-      (name = "indexOf" or name = "includes" or name = "startsWith" or name = "endsWith") and
+      (name = "indexOf" or name = "lastIndexOf" or name = "includes" or name = "startsWith" or name = "endsWith") and
```

Comment:
Should the `name = "indexOf"` on [line 30](https://github.com/Semmle/ql/pull/699/files#diff-6f6ebbf96fe90230682a46d1f9351631R30) be updated as well?

Reply:
Hmm, yes. Fixed.